### PR TITLE
Check where phantomjs is installed on travis

### PR DESCRIPTION
**Pull request by simong**

### CODE REVIEW

**Comment by simong** [comment]
_This is not the PR you are looking for_

**Comment by mrvisser** [comment]
👀

**Comment by mrvisser** [comment]
Discussed in Slack. Sending back to @simong as it's not clear that removing the directory v.s. the binary is fixing anything.

**Comment by simong** [comment]
Added a post install step that checks whether phantomjs is installed

File: .travis.yml [hunk 1]
```diff
@@ -55,7 +55,7 @@ before_install:
   - sudo service redis-server restart
 
   # We must ensure phantomjs is not available so the travis build will include it in node_modules when deploying
-  - sudo rm -rf /usr/local/phantomjs
+  - sudo rm -rf /usr/local/phantomjs/bin/phantomjs
```
Comment (mrvisser) [inline .travis.yml]: Is this change still needed / optimal?

**Comment by mrvisser** [comment]
Looks good, just a minor question. Feel free to merge once you have a look.

**Comment by simong** [comment]
Took the change back out and merged just the test check
